lib/date.ts

```diff
@@ -0,0 +1,90 @@
+/// <reference types="@nextcloud/typings" />
+
+declare var window: Nextcloud.v16.WindowWithGlobals
+	| Nextcloud.v17.WindowWithGlobals
+	| Nextcloud.v18.WindowWithGlobals
+	| Nextcloud.v19.WindowWithGlobals;
```

Comment:
Let's use [more recent typings](https://github.com/nextcloud/nextcloud-typings/pull/178)?

Reply:
ok, but does not change much (window has not changed) :)

Reply:
Can we do that in a followup then please?
For code consistency, it does make sense to upgrade those typing :)